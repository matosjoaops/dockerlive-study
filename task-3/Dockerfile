FROM node

RUN apt-get update && apt-get install -y nodejs

COPY . .

CMD node index.js