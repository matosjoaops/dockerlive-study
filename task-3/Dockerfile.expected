FROM node

COPY . .

RUN npm install --only=production

CMD node index.js